src/views/pie.js

```diff
@@ -0,0 +1,66 @@
+  let pieChart = document.getElementById("myChart");
+  let xValues = ["Flask", "React.js", "Unity", "USP.NET Core", "Laravel", "Drupal", "OWL(Odoo Web Library)", "Odoo", "Spring", "jQuery", "Vue.js", "Angular.js", "Angular", "Express", "FastAPI", "Django"];
+  let yValues = [19, 7, 4, 4, 4, 4, 4, 4, 4, 4, 7, 7, 11, 4, 7, 7,];
```

Comment:
Now this data should be coming from the [survey.yaml](https://github.com/osscameroon/devsurvey/blob/main/src/res/survey.yaml) file. We should not write them ourselves.